### ISSUE TITLE
Change player input

### ISSUE THREAD

**Issue by dissolve** [dissolve/PieStation#2]
@meleu requested
A simple way to choose the input for player 1, 2, 3 and 4. Maybe you can borrow some code/get inspiration from here.
https://github.com/Herdinger/EmulationStation/blob/master/es-core/src/InputManager.cpp

**Comment by meleu** [comment]
It looks like this (from recalbox system):
![alt text](https://raw.githubusercontent.com/recalbox/recalbox-os/master/wiki/images/bluetooth-pair.jpg)

**Comment by dissolve** [comment]
Hmm, whats the options like on that?  seems a bit odd.  I would have thought it better to have just a re-sorting of inputs.

**Comment by meleu** [comment]
yeah! please, ignore the "pair a bluetooth controller option". my request is just for re-sorting the inputs. On the image, in the fields with "DEFAULT", if hit right/left, it loops through the available joysticks.
I made an script to achieve this goal at least for RetroArch emulators (as you can see [here](https://github.com/meleu/RetroPie-joystick-selection)), but I would be happy to see this feature directly in emulationstation.

**Comment by meleu** [comment]
@dissolve 
I would like to reinforce an argument for the implementation of this feature: I have many friends that choose Recalbox over RetroPie because of this feature.

They have several controllers plugged on raspi, when they want to play arcade games, they use arcade-like controllers. When they want to play the old console games, they use a regular joystick. That's why I made a [script to simplify this configuration](https://github.com/meleu/RetroPie-joystick-selection/) (at least for RetroArch emulators), but it would be really nice to see it in emulationstation.

**Comment by dissolve** [comment]
@meleu oh I agree, this is certainly a really useful feature to have. :+1:
